ext/ffi/lib.rs

```diff
@@ -2044,23 +2008,24 @@ where
     } else {
       &backing_store[..] as *const _ as *const u8
     }
-  } else if let Ok(value) = v8::Local::<v8::ArrayBuffer>::try_from(buf.v8_value)
+  } else if let Ok(value) =
+    v8::Local::<v8::ArrayBuffer>::try_from(buf_or_number)
   {
     let backing_store = value.get_backing_store();
     &backing_store[..] as *const _ as *const u8
+  } else if let Ok(value) = v8::Local::<v8::BigInt>::try_from(buf_or_number) {
+    value.u64_value().0 as usize as *const u8
+  } else if let Ok(value) = v8::Local::<v8::Number>::try_from(buf_or_number) {
+    value.value() as usize as *const u8
   } else {
     return Err(type_error(
       "Invalid FFI buffer, expected ArrayBuffer, or ArrayBufferView",
```

Comment:
Error is out of sync with the parameter type? I.e. it now allows number/bigint

Reply:
Will fix if we decide to move forward with this PR.